frontend/__mocks__/crds.ts

```diff
@@ -16,15 +16,13 @@ export const testCRD: CustomResourceDefinitionKind = {
   spec: {
     group: 'testapp.coreos.com',
     version: 'v1alpha1',
```

Comment:
I think if we update the CRD API version to v1, we have to update this to use `versions` instead of `version`. The version property is removed in v1.
```suggestion
    versions: [
      {
        name: 'v1alpha1',
        storage: true,
        served: true,
        schema: {
          openAPIV3Schema: {}
        }
```